pmg/views.py

```diff
@@ -961,9 +961,13 @@ def members():
                 'sphere'] == 'national':
             members_by_house.setdefault(member['house']['name'],
                                         []).append(member)
+    id_mapping = {
+        house: house.lower().replace(" ", "-") for house in members_by_house.keys()
```

Comment:
isn't there a nice slugify function at hand so you don't need to slugify manually?

Reply:
Nah, flask doesn't have one.
There are python libraries that can do it but that would mean installing a dependency.

Reply:
ja, I've used https://github.com/un33k/python-slugify - I think adding this as a dep is better than a brittle hack - we don't want this to break because someone decided to put something random into a name